src/eslint-plugin-sx/docs/rules/no-unused-stylesheet.md

```diff
@@ -53,4 +53,4 @@ _none_
 
 ## When Not To Use It
 
-TKTK
+There should be no valid reason to turn this rule off. It helps with a dead code elimination.
```

Comment:
> When Not To Use It

`If you are not using sx` instead maybe?

Reply:
Well, it's inside eslint-plugin-sx so that would relate to the whole package. Similarly "if you are not using Eslint". :) In my eyes this rather describes the single rule usage when already using SX with this plugin.

Reply:
Good point 😂